src/modules/tasks/components/TasksList/TasksList.tsx

```diff
@@ -30,178 +32,216 @@ export interface getTaskTypes {
     description: string;
   };
 }
-
+interface getTasks {
+  pageNumber: number;
+  pageSize: string;
+  data: getTaskTypes[];
+}
 const TasksList = () => {
-const [selectedId, setSelectedId] = useState<number>(0);
-const [view, setView] = useState<boolean>(false);
-const [showDelete, setShowDelete] = useState(false);
-const handleCloseDetails = () => setView(false);
-const handleClose = () => setShowDelete(false);
-const [arrayOfPages, setArrayOfPages] = useState<number[]>([]);
-const [numOfRecords, setNumOfRecords] = useState<number>(0);
-const [pageNum, setPageNum] = useSearchParams();
-
-const handleView = (id: number) => {
-  setSelectedId(id);
-  setView(true);
-};
-const deleteTask = async () => {
-  try {
-    const response = await axiosInstance.delete(TASKS_URLS.DELETE_TASK(1015));
-    if (response?.data.affected !== 0) {
-      toast.success("Task deleted successfully");
-      // taskQuery?.triggerTasks(params.get("page") || 1);
-      // getTasks();
-    } else {
-      toast.error("Task not found");
-    }
-  } catch (error) {
-    if (error instanceof AxiosError) {
-      if (error.response) {
-        toast.error(error.response.data.message || "something went wrong");
+  const [selectedId, setSelectedId] = useState<number>(0);
+  const [view, setView] = useState<boolean>(false);
+  const [showDelete, setShowDelete] = useState(false);
+  const handleCloseDetails = () => setView(false);
+  const handleClose = () => setShowDelete(false);
+  const [arrayOfPages, setArrayOfPages] = useState<number[]>([]);
+  const [numOfRecords, setNumOfRecords] = useState<number>(0);
+  const [pageNum, setPageNum] = useSearchParams();
```

Comment:
We can use the hook once and then get the values we want. I don't reccomend naming the values as `[pageNum, setPageNum]` as it will be misleading for any one reading the code.